backend/generator.py

```diff
@@ -35,11 +35,13 @@ class InferenceConfig:
 
     num_inference_steps: int = 100
     controlnet_conditioning_scale: float = 1.5
-    guidance_scale: float = 8.0
-    negative_prompt: str = "ugly, disfigured, low quality, blurry"
+    control_guidance_start: float = 0.25
```

Comment:
Offline, you indicated that these parameters were resulting in a large fraction of QR codes that scanned (a scannability rate of ~0.75).

Since we've got a trustworthy scan eval, I think we want to set the parameters such that the scannability rate is more like ~0.3, so that we end up with a scannable code ~95% of the time with batch size eight.